src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -617,6 +616,11 @@ def generate_MCP_1_default_config(self, filename):
                                                                  pal_low_frequency,
                                                                  pal_high_frequency,
                                                                  'test_user_1')
+
+    # Add a CBSD to be part of the PPA
+    cbsd_reference_id1 = generateCbsdReferenceId('test_fcc_id_x', 'test_serial_number_x')
```

Comment:
What is this for?

Reply:
The ppa record in testdata folder does not have ‘cbsdReferenceId’ field, here we are creating the field and adding dummy reference id to it. 
This code is added for testing IAP and Aggregate models. If we are going to add the field in input ppa record, we can ignore the change.
Please let us know your thoughts.